src/explorer/appServiceNode.ts

```diff
@@ -85,6 +85,34 @@ export class AppServiceNode extends NodeBase {
         return this.start();
     }
 
+    async delete(azureAccount: AzureAccountWrapper): Promise<boolean> {
+        let servicePlanName = this.site.serverFarmId.substring(this.site.serverFarmId.lastIndexOf('/serverfarms/') + '/serverfarms/'.length);
+        let servicePlanRG = this.site.serverFarmId.substring(this.site.serverFarmId.indexOf('resourceGroups/') + 'resourceGroups/'.length, this.site.serverFarmId.indexOf('/providers/'));
+        let servicePlan = await this.getWebSiteManagementClient(azureAccount).appServicePlans.get(servicePlanRG, servicePlanName);
+        let options = {};
+        options['deleteEmptyServerFarms'] = false;
+        let mOptions: MessageOptions = { modal: true };
+        if (servicePlan.numberOfSites < 2) {
+            let input = await window.showInformationMessage(`This is the last web app on the plan, "${servicePlanName}".  Delete the Web App and App Service Plan?`, mOptions, ...['Both', 'Web App ONLY']);
```

Comment:
Instead of `...['Both', 'Web App ONLY']` you can just do `'Both', 'Web App ONLY'`

Reply:
Do you think showWarningMessage is better for operations like delete?